projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -905,11 +902,6 @@ function wpcom_update_about_page_check( $post_id, $post ) {
 		return;
 	}
 
-	// We only care about published pages. Pages added via the API are not published by default.
-	if ( $post->post_status !== 'publish' ) {
-		return;
-	}
```

Comment:
Could you explain why we're running the check for all post statuses? Do we want to count any basic changes that are not published as completing the task? I want to make sure I understand the intent here, as I think this may have impact on the performance for saving drafts across WordPress.com sites.

Reply:
> Late-breaking thought: do we want to limit this to published posts? I am not sure we do, as modifying an unpublished page is still a good thing. If we want to add the filter, we'd want to add 'post_status' => 'publish' to the filter.

Did I misunderstand your comment about not limiting to only published posts? The implication to me was that applied in this location _and_ the call to `get_posts`.

Reply:
My thinking wasn't that comprehensive! But when I saw this check removed, I thought it through a bit, and I think I have the following thoughts:
* When we're looking for an About page to update, we are wanting to find a post created by Headstart, so we're not too worried about what publish state it is in to start with.
* When we're trying to detect task completion, we want to know that the user updated that previously identified post. I was thinking that we specifically want them to publish their changes, not just tinker with the content and trigger an auto-save or save without a publish. I suppose we could accept deleting the post, too, as I think that's a valid (and mostly terminal!) update!

I hope that clarifies my thinking, @markbiek!

Reply:
That makes total sense. Sorry, I think I was thinking about it too hard 😝 . I've added this check back in.